unbalanced_dataset/under_sampling/instance_hardness_threshold.py

```diff
@@ -0,0 +1,240 @@
+"""Class to perform under-sampling based on the instance hardness
+threshold."""
+from __future__ import print_function
+from __future__ import division
+
+import numpy as np
+
+from collections import Counter
+
+from scipy.stats import mode
+
+from sklearn.utils import check_X_y
+from sklearn.neighbors import NearestNeighbors
+from sklearn.cross_validation import StratifiedKFold
```

Comment:
You should used the new module `model_selection`: [docs](http://scikit-learn.org/dev/modules/generated/sklearn.model_selection.StratifiedKFold.html#sklearn.model_selection.StratifiedKFold)

The function that you are using will be deprecated in the future. We can anticipate this from now ;)